packages/neutrino/src/webpack/develop.js

```diff
@@ -0,0 +1,31 @@
+const ora = require('ora');
+const merge = require('deepmerge');
+const webpack = require('webpack');
+const DevServer = require('webpack-dev-server');
+
+module.exports = _config => new Promise((resolve) => {
+  const config = merge({}, _config);
```

Comment:
Maybe this could be consolidated to some defaults:

```js
const config = merge({
  devServer: {
    https: false,
    host: 'localhost',
    port: 5000,
    noInfo: true
  }
}, _config);
const protocol = config.devServer.https ? 'https' : 'http';
const { host, port } = config.devServer;
const url = `${protocol}://${host}:${port}`;
```